mongodb/src/main/scala/akka/stream/alpakka/mongodb/Implicits.scala

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2015 MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package akka.stream.alpakka.mongodb
+
+import java.util.concurrent.atomic.AtomicBoolean
+
+import org.mongodb.{scala => mongoDB}
+import org.{reactivestreams => rxStreams}
+
+import scala.language.implicitConversions
+
+object Implicits {
+
+  implicit def observableToPublisher[T](observable: mongoDB.Observable[T]): rxStreams.Publisher[T] = ObservableToPublisher(observable)
```

Comment:
This is used in one place, I don't think that warrants using implicits, just use the adapter explicitly instead. Also, it should be private to the mongodb connector and not public API